docs/os/windows/windows-vm41.md

```diff
@@ -0,0 +1,203 @@
+Installing a Windows VM
+=======================
+
+Simple Windows install
+----------------------
+
+If you just want something simple and you can live without some features. This works for Windows XP, 7, 10 and 11, and it may work for Windows 8 and 8.1, although this has not been tested.
+
+**Works:**
+- display (1440x900 or 1280x1024 are a nice fit onto FHD hw display)
+- keyboard (incl. correct mapping), pointing device
+- network (emulated Realtek NIC)
+- audio output and input (available even without QWT installation if `qvm-features audio-model` is set as `ich6`)
+
+**Does not work:**
+- copy & paste (the qubes way)
+- copying files into / out of the VM (the qubes way)
+- assigning USB devices (the qubes way via the tray applet)
+- PCI device 5853:0001 (Xen platform device) - no driver
+- all other features/hardware needing special tool/driver support
+
+**Installation procedure:**
+- Have the Windows ISO image (preferrably the 64-bit version) downloaded in some qube.
```

Comment:
maybe add the forum's wiki section about where to obtain isos (both official iso + "tweaked) ?
+ maybe a link to the issue where you discuss signing ?